conf/modules.config

```diff
@@ -238,6 +238,12 @@ process {
             ext.args2        = '-bS'
             ext.prefix       = {"${meta.id}.umi_unsorted"}
         }
+// TODO: args1? args2 should be good
+        withName: ".*DRAGMAP_ALIGN" {
+            ext.args         = { "-p -C -M -R ${meta.read_group}" }
```

Comment:
these are bwa parameters, no? should be `--RGSM ${meta.read_group}`i believe, see below line 268